sklearn/impute.py

```diff
@@ -141,6 +141,22 @@ class SimpleImputer(BaseEstimator, TransformerMixin):
     statistics_ : array of shape (n_features,)
         The imputation fill value for each feature.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.impute import SimpleImputer
+    >>> X = [[np.nan, 2, 3], [4, np.nan, 6], [10, np.nan, 9]]
+    >>> imp_mean = SimpleImputer(missing_values=np.nan, strategy='mean')
+    >>> imp_mean.fit([[7, 2, 3], [4, np.nan, 6], [10, 5, 9]])
```

Comment:
use X as defined above.

Reply:
ok I see now that you use it in transform. I will move the definition just before using it.

Reply:
OK, thanks, will do!

Reply:
It's done. I pushed to your branch.